src/ipfs.io/docs/install/index.html

```diff
@@ -10,118 +10,91 @@
 <div class="col-lg-6 col-lg-offset-3 col-sm-8 col-sm-offset-2" style="font-family: Oxygen; line-height: 1.7">
 {% markdown %}
 
-# Install IPFS
+# Install Go IPFS
 
-You can install the [Go IPFS implementation](https://github.com/ipfs/go-ipfs)
-from source or from prebuilt packages.
+We recommend to install IPFS from a prebuilt package. You can obtain those from [dist.ipfs.io](http://dist.ipfs.io/#go-ipfs).
 
+For convenience here are the latest versions (`0.3.11`) linked directly
+- <i class="fa fa-apple"></i> [Mac OS X]
+- <i class="fa fa-linux"></i> [Linux 32bit]
+- <i class="fa fa-linux"></i> [Linux 64bit]
+- <i class="fa fa-windows"></i> [Windows 32bit]
+- <i class="fa fa-windows"></i> [Windows 64bit]
 
-## Prebuilt Packages
+## Installing from a Prebuilt Package
 
-We use [gobuilder.me](https://gobuilder.me) to do automated builds.
+### Mac OS X and Linux
 
-Download a binary for your platform (more coming):
+After downloading, untar the archive, and move `ipfs` somewhere in your `$PATH`:
 
-- <i class="fa fa-apple"></i> [Mac OS X](https://gobuilder.me/get/github.com/ipfs/go-ipfs/cmd/ipfs/ipfs_release_darwin-amd64.zip)
-- <i class="fa fa-linux"></i> [Linux x86](https://gobuilder.me/get/github.com/ipfs/go-ipfs/cmd/ipfs/ipfs_release_linux-386.zip)
-- <i class="fa fa-linux"></i> [Linux x86_64](https://gobuilder.me/get/github.com/ipfs/go-ipfs/cmd/ipfs/ipfs_release_linux-amd64.zip)
-- <i class="fa fa-linux"></i> [Linux ARM](https://gobuilder.me/get/github.com/ipfs/go-ipfs/cmd/ipfs/ipfs_release_linux-arm.zip)
-- [See all builds](https://gobuilder.me/github.com/ipfs/go-ipfs/cmd/ipfs?branch=release)
-
-Then, unzip the archive, and move `ipfs` somewhere in your `$PATH`:
-
-```
-mv ipfs /usr/local/bin/ipfs
+```sh
+$ tar xvfz go-ipfs_v0.3.11_darwin-amd64.tar.gz
+$ mv go-ipfs/ipfs /usr/local/bin/ipfs
```

Comment:
Let's avoid the `$` because it makes copy-pasting a bit annoying. this is one of the few places where we want to cut as many steps as we can.